### ISSUE TITLE
Other 7 carpal bone fractures

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10758]
**Preferred term label:**
In addition to https://hpo.jax.org/browse/term/HP:6001107 the 7 others are:

- [x] Fractured Lunate bone 
- [x] Fractured Triquetrum bone 
- [x] Fractured Pisiform bone	 	 
- [x] Fractured Trapezium bone	
- [x] Fractured Trapezoid bone	
- [x] Fractured Capitate bone	
- [ ] Fractured Hamate bone	

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0041248

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Each has a clinical picture that our orthopedics curator considers important to distinguish

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001304
- label: Fractured lunate bone
- parents: [Fractured carpal bone (HP:0041248)](https://hpo.jax.org/app/browse/term/HP:0041248)
- pmid: 
- def: A partial or complete breakage of the lunate bone.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10758
- synonyms: Lunate bone fracture(NONE)|Lunate fracture(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001305
- label: Fractured triquetrum bone
- parents: [Fractured carpal bone (HP:0041248)](https://hpo.jax.org/app/browse/term/HP:0041248)
- pmid: 
- def: A partial or complete breakage of the triquetrum bone.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10758
- synonyms: Triquetrum bone fracture(NONE)|Triquetrum fracture(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001306
- label: Fractured pisiform bone
- parents: [Fractured carpal bone (HP:0041248)](https://hpo.jax.org/app/browse/term/HP:0041248)
- pmid: 
- def: A partial or complete breakage of the pisiform bone.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10758
- synonyms: Pisiform fracture(NONE)|Pisiform bone fracture(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001307
- label: Fractured trapezium bone
- parents: [Fractured carpal bone (HP:0041248)](https://hpo.jax.org/app/browse/term/HP:0041248)
- pmid: 
- def: A partial or complete breakage of the trapezium bone.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10758
- synonyms: Trapezium bone fracture(NONE)|Trapezium fracture(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001308
- label: Fractured trapezoid bone
- parents: [Fractured carpal bone (HP:0041248)](https://hpo.jax.org/app/browse/term/HP:0041248)
- pmid: 
- def: A partial or complete breakage of the trapezoid bone.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10758
- synonyms: Trapezoid bone fracture(NONE)|Trapezoid fracture(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001309
- label: Fractured capitate bone
- parents: [Fractured carpal bone (HP:0041248)](https://hpo.jax.org/app/browse/term/HP:0041248)
- pmid: 
- def: A partial or complete breakage of the capitate bone.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10758
- synonyms: Capitate bone fracture(NONE)|Capitate fracture(NONE)

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001310
- label: Fractured hamate bone
- parents: [Fractured carpal bone (HP:0041248)](https://hpo.jax.org/app/browse/term/HP:0041248)
- pmid: 
- def: A partial or complete breakage of the hamate bone.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10758
- synonyms: Hamate bone fracture(NONE)|Hamate fracture(NONE)

**Comment by pnrobinson** [comment]
@MickeySegal added all seven terms.
